meinberlin/apps/budgeting/assets/SupportBox.jsx

```diff
@@ -66,7 +78,7 @@ export const SupportBox = (props) => {
   return (
     <div className="rating">
       <button
-        aria-label={translations.support}
+        aria-label={support + ' ' + translations.support + getSupportClickStr()}
```

Comment:
so i'm a little torn here, I'm not sure if it's better to use a visually hidden span here instead of aria label? either way as it is this will read the count twice so we should try to not confuse the sr user so we should ensure it's read correctly

Reply:
i See, in Voiceover it does not read out the number twice. But added so it will definitely skip the second number to be read out.

Reply:
I just added a comment in a4 about aria-hidden on spans but have not tested any of these as still can't install so maybe see if its working in Narrator, JAWs and NVD as they are the most used SRs, if it's fine we can leave it as is but if not then we should re-asses aria-hidden on non-semantic elements

Reply:
@phillimorland good point as I never tried another SR before. I did now and for JAWs and NVDA it works the set aria-hidden on a <kbd><span></kbd>.

Also on MDN they use a `span` as an example (even if they use the span for an icon):
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden#example